packages/light-client/test/unit/webEsmBundle.browser.test.ts

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-call */
```

Comment:
This just removes diff from electra branch, this file shouldn't be modified there

Reply:
But my PR adds the removed eslint disable directive again, we shouldn't be modifying `packages/light-client/test/unit/webEsmBundle.browser.test.ts` on the electra branch at all.

This makes no difference as we completely ignore this file from eslint since recently but less diff on electra branch is a good thing.